admin/commands/inventory/add_agent_qan_postgres_pgstatements_agent.go

```diff
@@ -57,7 +57,7 @@ type AddAgentQANPostgreSQLPgStatementsAgentCommand struct {
 	TLSCAFile           string            `name:"tls-ca-file" help:"TLS CA certificate file"`
 	TLSCertFile         string            `help:"TLS certificate file"`
 	TLSKeyFile          string            `help:"TLS certificate key file"`
-	LogLevel            string            `enum:"debug,info,warn,error,fatal" default:"warn" help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
+	LogLevel            string            `enum:"debug,info,warn,error" default:"warn" help:"Service logging level. One of: [debug, info, warn, error]"`
```

Comment:
QAN agents are not related to postgres_exporter

Reply:
done